modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,87 @@
+= Policies Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies differ based on several different factors, such as category, purpose, version, and configuration options. You must carefully review these factors, before you decide to implementation them in your environment. In this section, each of these factors are explained in detail. 
+
+== Policy Architecture
+
+Policies are implemented through coordinated communication between the following components:
+
+* API Manager
+* One or more API Gateway Runtimes, or Mule 3.8.0 Runtime or later
+* One or more API proxy or API applications
+
+Using API Manager, you can configure and apply policies to an API instance. An API version can have one or more API instances. +
+A specific API instance can also be tied to a specific API implementation endpoint, which can then auto-generate an API proxy application. +
+You can deploy an API Proxy to a Mule runtime.
+
+Each API proxy application receives requests on HTTP or HTTPS URLs specified by the API. Normally requests are forwarded to the corresponding API implementation endpoint URL, and then the response travels through the API proxy application back to the requesting client application.
+
+== Policy Injection and Enforcement
+
+When you deploy an API proxy application for a specific API version to a Mule Runtime, any applied policies in the platform, or offline policies in the runtime, are injected into the API proxy. This changes the behavior of the application. When the proxy application receives a new request, all injected policies are applied to decide if, and how, the request is forwarded to the API implementation endpoint.
+
+In this way, the actual policy enforcement occurs inside the proxy application itself, minimizing the cross talk between the API proxy, which is processing the received request, the Anypoint Platform agent, and online API Manager. The API proxy application does not require any communication with the Anypoint Platform agent running in the runtime, nor does it require any communication with API Manager.
+
+However, the Anypoint Platform agent remains connected to API Manager. After policies are reconfigured or removed from API Manager, those policies are downloaded to any connected API Gateway or Mule runtimes, which updates each runtime /policies folder. The policy changes are then again injected into each API proxy application. This allows policies to be dynamically changed without having to redeploy the API proxy application, and without having to restart the API Gateway or Mule runtimes.
```

Comment:
GLOBALLY lowercase "gateway" in "API gateway."
```suggestion
However, the Anypoint Platform agent remains connected to API Manager. After policies are reconfigured or removed from API Manager, those policies are downloaded to any connected API gateway or Mule runtime engines, which updates each runtime /policies folder. The policy changes are then again injected into each API proxy application. This allows policies to be dynamically changed without having to redeploy the API proxy application and without having to restart the API gateway or Mule runtime engines.
```

Reply:
Done! Thanks!